networks/subsocial/package.json

```diff
@@ -21,8 +21,8 @@
       "lodash": "^4.17.21"
     },
     "devDependencies": {
-      "@subsocial/types": "^0.6.6",
-      "@polkadot/api": "^7",
+      "@subsocial/types": "^0.6.8",
+      "@polkadot/api": "^8",
```

Comment:
As I see subql/node supports only polkadot/api 7.14.3. Will it work correctly with this version? polkadot api 8 doesn't have usage changes, but it need compatible libraries, and I think we need double check for it

Reply:
I changed it because the Subsocial project was not building with an error:

<details>
  <summary>Comparison</summary>

<img width="1172" alt="Screenshot 2022-04-11 at 13 34 47" src="https://user-images.githubusercontent.com/40560660/162723120-e58de460-925a-4653-8fc9-1f1b934ee988.png">

<img width="1163" alt="Screenshot 2022-04-11 at 13 36 15" src="https://user-images.githubusercontent.com/40560660/162723144-2481b818-e77b-48ae-8b16-5ad9659a5fef.png">


</details>